src/solidity-frontend/solidity_convert.cpp

```diff
@@ -846,6 +846,12 @@ bool solidity_convertert::get_expr(
         return true;
       break;
     }
+    case SolidityGrammar::ElementaryTypeNameT::STRING_LITERAL:
+    {
+      if(convert_string_literal(literal, the_value, new_expr))
```

Comment:
Add a comment to explain this line? Perhaps, "converting the RHS of `string s = "HELLO"`"?

Reply:
I believe the method name is self explanatory when it comes to describing the purpose of this statement. I should, however, add doc comments to the method itself to better explain what it does